plugins/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageClientSettings.java

```diff
@@ -66,13 +94,23 @@
 
     /** Name used by the client when it uses the Google Cloud JSON API. **/
     static final Setting.AffixSetting<String> APPLICATION_NAME_SETTING = Setting.affixKeySetting(PREFIX, "application_name",
-        key -> new Setting<>(key, "repository-gcs", s -> s, Setting.Property.NodeScope));
+            key -> new Setting<>(key, "elasticsearch-repository-gcs", Function.identity(), Setting.Property.NodeScope,
```

Comment:
Ok. Originally, I had taken the liberty to change this because it is prefixed internally by the library and appends the version too, and I though it's not that important, although who I'm to say it might not get untangled on the server side.